participants/sophia_fairbairn.json

```diff
@@ -0,0 +1,43 @@
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your name / nickname (required)
+	"name": "Sophia Fairbairn",
+	// optional company name (required)
+	"company": "actively seeking opportunities",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// 2023-06-30
+		"friday": true,
+		// 2023-07-01
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["React", "TypeScript", "WebScraping", "Node.JS", "Vite"],
+	// if you only eat vegan food (optional)
+	"vegan": null,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": null,
+	// what you cannot eat or drink (optional)
+	"allergies": null,
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "JavaScript has had a profound impact on my journey as someone transitioning into software development. Learning JavaScript has opened up a world of possibilities and opportunities for me in the software development field.",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "volunteer support, content creation, taking notes, and organizational skills - as well as a more business oriented perspective",
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	"tShirt": {
+		// S | M | L | XL | 2XL | 3XL
+		"size": "S",
+		// fitted (also known as waist cut or women variant) or regular
+		"type": "fitted" 
+	},
+	
+	// your Twitter handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"twitter": "Sophia_techy",
+	// your Mastodon URL (optional)
+	"mastodon": null,
+	// your website URL or other social media (optional)
+	"website": "https://www.your-domain.com"
```

Comment:
Please leave this out if you're don't have a link to share on the participants list or replace it with a real page - thanks!

```suggestion
```